db/system_distributed_keyspace.cc

```diff
@@ -349,7 +349,7 @@ future<std::unordered_map<utils::UUID, sstring>> system_distributed_keyspace::vi
             db::consistency_level::ONE,
             internal_distributed_query_state(),
             { std::move(ks_name), std::move(view_name) },
-            false).then([this] (::shared_ptr<cql3::untyped_result_set> cql_result) {
+            cql3::query_processor::cache_internal::no).then([this] (::shared_ptr<cql3::untyped_result_set> cql_result) {
```

Comment:
Perhaps we should just eliminate the parameter. Why not cache it? If it is not reused, it will naturally drop out of the cache.

Reply:
Ah, in the next patch I see internal queries are cached with no TTL. Perhaps we should fix that and then cache everything.

Reply:
Our internal cache is a simple map from string to a prepared statement. It doesn't have expiration times like our user
queries cache and the internal queries are cached forever. I thought of changing this map to a cache eventually but this seamed like a first logical step.